app/inpage/ts/inpage.ts

```diff
@@ -377,26 +370,16 @@ class InterceptorMessageListener {
 			})
 			if (forwardRequest.requestId !== undefined) this.outstandingRequests.get(forwardRequest.requestId)!.resolve(reply)
 		} catch (error: unknown) {
-			// if it is an Error, add context to it if context doesn't already exist
-			console.log(error)
-			console.log(messageEvent)
 			if (forwardRequest.requestId === undefined) return
-			if (error instanceof Error) {
-				const inputedErrorObject = {
-					...error,
-					code: ('code' in error && typeof error.code !== 'number') ? error.code : -32603,
-					data: !('data' in error) || error.data === undefined || error.data === null || typeof error.data !== 'object' ? { request: forwardRequest } : { ...error.data, request: forwardRequest }
-				}
-				return this.outstandingRequests.get(forwardRequest.requestId)!.reject(inputedErrorObject)
-			}
+			if (error instanceof Error) return this.outstandingRequests.get(forwardRequest.requestId)!.reject(error)
```

Comment:
If this suggestion doesn't work, then a comment should be included indicating how you are certain it won't be undefined despite the compiler not being certain.
```suggestion
			if (error instanceof Error) return this.outstandingRequests.get(forwardRequest.requestId).reject(error)
```